modules/api/pkg/api/v2/types.go

```diff
@@ -2004,8 +2009,8 @@ type SeedStatus struct {
 	Phase kubermaticv1.SeedPhase `json:"phase"`
 }
 
-// GlobalSettings defines global settings
-// swagger:model GlobalSettings
+// GlobalSettingsV2 defines global settings
+// swagger:model GlobalSettingsV2
```

Comment:
Why v2? Do we have same type in `api/v1` and swagger fails?

Reply:
Yeah, it's failing for everyone, and everyone just `retries` till it passes. Which it eventually does because there is a race condition, sort of, between 2 different specs for the same resource.

Reply:
But I think we need to change the struct name as well for properly fixing it

Reply:
Turns out, we don't need the v1 struct anyways.